src/style/component.scss

```diff
@@ -0,0 +1,151 @@
+.App {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  .circle-inner {
+    position: fixed;
+    width: 5px;
+    height: 5px;
+    border-radius: 2.5px;
+    background-color: black;
+    transition: 0.1s;
+    pointer-events: none;
+  }
+  .circle-outer {
+    position: fixed;
+    width: 20px;
+    height: 20px;
+    border-radius: 10px;
+    background-color: transparent;
+    border: 2px solid gray;
+    transform: translate(-7.5px, -7.5px);
+    transition: 0.2s;
+    pointer-events: none;
+  }
+  .App-header{
+    position: fixed;
```

Comment:
can you tell me the reason why you upper case the first letter of classname?

Reply:
the default classname is App and I add -header :)

Reply:
`.app-header` will do, don't have a convention for upper case first letter for className